conan/tools/cmake/toolchain/blocks.py

```diff
@@ -328,8 +328,64 @@ class AppleSystemBlock(Block):
         # Setting CMAKE_OSX_DEPLOYMENT_TARGET if "os.version" is defined by the used conan profile
         set(CMAKE_OSX_DEPLOYMENT_TARGET "{{ CMAKE_OSX_DEPLOYMENT_TARGET }}" CACHE STRING "")
         {% endif %}
+        {% if enable_bitcode is defined and enable_bitcode is sameas true %}
+        # Bitcode
+        set(CMAKE_XCODE_ATTRIBUTE_ENABLE_BITCODE YES)
+        set(CMAKE_XCODE_ATTRIBUTE_BITCODE_GENERATION_MODE bitcode)
+        {% endif %}
+        {% if enable_arc is defined and enable_arc is sameas true %}
+        # ARC
+        set(CMAKE_XCODE_ATTRIBUTE_CLANG_ENABLE_OBJC_ARC YES)
+        {% elif enable_arc is defined and enable_arc is sameas false %}
+        # ARC
+        set(CMAKE_XCODE_ATTRIBUTE_CLANG_ENABLE_OBJC_ARC NO)
```

Comment:
Feels like 2 PR in one. One is the addition of bitcode and other flags, and the other the injection of general flags. But it is fine, it was just a bit confusing at first.